@here/harp-mapview/lib/text/Placement.ts

```diff
@@ -617,6 +631,8 @@ function placePointLabelAtAnchor(
         tmpMeasurementParams.letterCaseArray = label.glyphCaseArray!;
         // Compute label bounds according to layout settings.
         textCanvas.measureText(label.glyphs!, label.bounds, tmpMeasurementParams);
+        // TODO: Make the margin configurable
+        label.bounds.expandByVector(pointLabelMargin);
```

Comment:
The biggest question mark is here. Should we apply the margin before or after scaling... Both cases has some strong and weak arguments.
- If we apply margin before scaling (margin is scaled together with other dimensions - font size, offsets) it is directly related to other sizes configured, but it is hard to say what is its final measure on the screen.
-If we apply margin after scaling is done on bounding box, then margin stays always the same (in screen space), but is not related to other measures and sizes. This approach will require performing more sophisticated logic, like with text offset, cause we need to know in which direction margin moves the text.

Reply:
with your change, the margin contributes to the text offset. And since we want the offset to scale, then it also makes sense that the margin does.
So it's ok as you have it here right now.